From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Matthias Maennich <maennich@google.com>
Date: Mon, 24 Jun 2019 18:10:19 +0100
Subject: ANDROID: gki/cuttlefish defconfigs: use prebuilt build-tools

Rather than relying on the hosts build tools, use a set of prebuilt
build-tools. These are tools like awk, cat, cp, hostname, python, rm,
sed, etc.

Bug: 135922132
Change-Id: Ie6b0bd208c684c9f153eb2f0bfac4712e02e8e05
Signed-off-by: Matthias Maennich <maennich@google.com>
---
 build.config.gki.aarch64 | 1 +
 build.config.gki.x86_64  | 1 +
 2 files changed, 2 insertions(+)

diff --git a/build.config.gki.aarch64 b/build.config.gki.aarch64
index ecc2985e92cb..24dade721f96 100644
--- a/build.config.gki.aarch64
+++ b/build.config.gki.aarch64
@@ -8,6 +8,7 @@ EXTRA_CMDS=''
 KERNEL_DIR=common
 POST_DEFCONFIG_CMDS="check_defconfig"
 CLANG_PREBUILT_BIN=prebuilts-master/clang/host/linux-x86/clang-r353983c/bin
+BUILDTOOLS_PREBUILT_BIN=prebuilts/build-tools/path/linux-x86
 LINUX_GCC_CROSS_COMPILE_PREBUILTS_BIN=prebuilts/gcc/linux-x86/aarch64/aarch64-linux-android-4.9/bin
 FILES="
 arch/arm64/boot/Image.gz
diff --git a/build.config.gki.x86_64 b/build.config.gki.x86_64
index 69d339dce4a0..3ce45e962732 100644
--- a/build.config.gki.x86_64
+++ b/build.config.gki.x86_64
@@ -8,6 +8,7 @@ EXTRA_CMDS=''
 KERNEL_DIR=common
 POST_DEFCONFIG_CMDS="check_defconfig"
 CLANG_PREBUILT_BIN=prebuilts-master/clang/host/linux-x86/clang-r353983c/bin
+BUILDTOOLS_PREBUILT_BIN=prebuilts/build-tools/path/linux-x86
 LINUX_GCC_CROSS_COMPILE_PREBUILTS_BIN=prebuilts/gcc/linux-x86/x86/x86_64-linux-android-4.9/bin
 FILES="
 arch/x86/boot/bzImage
